faqs/galaxy/histories_unshare.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Find Histories that have been shared with you, and unshare those not needed
+area: histories     
+box_type: tip        
+layout: faq        
+contributors: [nurzhamalyrys] 
+---
+
+1. All account Histories owned by others, but shared with you, can be reviewed under User > Histories shared with me.
+2. "Histories shared with me" is part of your own quota usage, and unsharing will not significantly reduce quota usage for most end-users. However, if there are hundreds (or more!), clearing these up periodically (unsharing) could be worth it.
```

Comment:
This is not what is written in the original FAQ [Find Histories that have been shared with you, and unshare those not needed](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed). Your sentence is not very clear and may be confusing for readers. 
In the original FAQ, we have:
*Only a small fraction of the data content in "Histories shared with me" are part of your own quota usage,*